### DISCUSSION TITLE
Dropped Frames and Rendering Lag While Using OBS-NDI/DistroAV

### DISCUSSION THREAD

**Discussion by JunderscoreB** [DistroAV/DistroAV#1153] [General]
I'm having major issues with this plugin for about a week and a half. I've had to turn it off. Basically, it interferes with the streaming and causes dropped frames and rendering issues, see screenshot from OBS:

![Screenshot 2024-11-20 130718](https://github.com/user-attachments/assets/6d341e14-ed0b-4d38-9c34-35c0a7648183)

It was happening with OBS-NDI and after updating to the latest DistroAV, the exact same issue is occurring. 

I recently upgraded to Win 11, but it was working fine two weeks ago on Win 11, so I don't think that's the problem. Any help would be appreciated, but basically this means I can't use the plugin until this issue is fixed. Happy to share other details if you need it. Thanks!

**Comment by BitRate27** [comment]
Can you upload a log and provide the link here?

**Comment by RayneYoruka** [comment]
+2 Post a log and please read the OBS wiki https://obsproject.com/kb/encoding-performance-troubleshooting.

If OBS is lagging, the lag will be reflected on the plugin.

**Reply by JunderscoreB** [reply]
I've done all the things I could in this troubleshooting guide.

**Comment by JunderscoreB** [comment]
Here's a link to a snippet of the log:

https://pastebin.com/t8qyFWnc

**Comment by RayneYoruka** [comment]
Need a full log to be able to look in to it. If you have problems with your OBS setup please refer to the OBS help forums.

**Comment by JunderscoreB** [comment]
Ok fine, here's the full log file. Note that I'm currently recording a local stream so the end of the log may look uncomplete. 

https://pastebin.com/DpAT5itW

Also, I currently have DistroAV turned off because of the lagging issues. And throughout the log, any long streams or recordings I had the plugin turned off.

**Comment by RayneYoruka** [comment]
Start here. https://obsproject.com/tools/analyzer?log_url=https%3A%2F%2Fobsproject.com%2Flogs%2FMbIQlW20g8PTsqAa#logURL

**Comment by JunderscoreB** [comment]
So, after using that analyzer tool, basically DistroAV is overloading the GPU, not sure why this is happening now, when two weeks ago (and prior) everything was dandy. Are there settings in DistroAV that I can adjust? I don't need a high framerate sent to NDI; really 1 fps would be fine for my application.

![image](https://github.com/user-attachments/assets/c4e440db-9389-4f45-aa24-79329ffd23fa)

**Comment by RayneYoruka** [comment]
Not sure what do you intent to do with a gt 710 in 2024 more than to watch videos, the gpu is almost end of life and does not support Nvenc on the latest beta release of obs.

[As per OBS github page: ](https://github.com/obsproject/obs-studio/releases/tag/30.2.3 )_NVIDIA users may need to update their GPU drivers to 531.61 (Windows) / 530.41.03 (Linux) or newer._ 

The Nvidia page shows the latest driver for that graphics card is: 475.14.

**Comment by JunderscoreB** [comment]
I realize it's an old card; it is on its latest available driver. We are not gaming here but rather an event space that is live streaming with a two camera set up. I'm using DistroAV to send a feed to the 2nd camera so the cameraperson knows which camera is live.

**Comment by RayneYoruka** [comment]
If the cpu and gpu are overloading because of outside factors, it's not the fault of the plugin.

**Comment by JunderscoreB** [comment]
But it only ever happens when I use the plugin. I can't figure out any other way to troubleshoot it... suggestions would be appreciated.

**Comment by RayneYoruka** [comment]
The plugin relies on GPU/Cpu power. If the gpu isn't on the correct drivers it will simply not work. Please upgrade your system with a more up to date graphics card.

**Comment by BitRate27** [comment]
In addition to what RayneYorka says about the GPU being overloaded, your only NDI source is from ProPresenter. This is a 100Mbps source so you must make sure you are on a 1Gb network. You are also outputting both the Main and Preview to NDI. Is this on purpose? If you don't need the output, I suggest turning those off.

**Comment by JunderscoreB** [comment]
We are on 1.0 GBE. Normally we run ProPresenter but currently we aren't and still having issues. I noticed the Main and Preview being both on, so I shut one of them off but, again, it makes no difference.

**Comment by BitRate27** [comment]
When you aren't using ProPresenter, you can select "Disconnect when not used" in NDI properties, this is new in 6.0.0. This will basically make it so you have no NDI source. How are you viewing/using the NDI output of Main?

**Comment by JunderscoreB** [comment]
I have another laptop that is grabbing the NDI feed that the cameraperson uses. I can try that "disconnect when not used", thanks for letting me know about that; however, we normally use it so it's something we need to contend with. I'll try to shut it off for now and see what happens...

**Reply by JunderscoreB** [reply]
Just a follow-up. The "disconnect when not used" setting didn't make a difference.

**Comment by BitRate27** [comment]
I will also add that I have my stuff running on a 1GBE switch, however I noticed the network port on my docking station was amber (100Mbps). I moved to the NIC on the laptop and it went green. You may have already checked if your port is green, just thought I would throw that out there as it caused me quite a headache to figure out why my stream was freezing and slides stopped work every once in a while.
